roboteam_ai/src/skills/Skill.h

```diff
@@ -14,29 +14,27 @@ namespace ai {
 
 // forward declare control Utils
 namespace control {
-    class ControlUtils;
-    class PosVelAngle;
+class ControlUtils;
+class PosVelAngle;
 }
 
-
-
 /**
  * \class Skill
  * \brief Base class for all skills. Provides no additional functionality.
  */
 class Skill : public bt::Leaf {
     private:
         roboteam_msgs::RobotCommand rotateRobotCommand(roboteam_msgs::RobotCommand &cmd);
-protected:
-        io::IOManager ioManager = io::IOManager(false,true);
-
+    protected:
+        using Control = control::ControlUtils;
```

Comment:
I would make it cutils instead of Control, which is confusing with the other namespace; 'control'

Reply:
Copied from Skill.h ;-)